Source/API/EbSvtVp9Enc.h

```diff
@@ -353,7 +353,7 @@ typedef struct EbSvtVp9EncConfiguration
 * @ *p_app_data      Callback data.
 * @ *config_ptr     Pointer passed back to the client during callbacks, it will be
 *                  loaded with default params from the library. */
-EB_API EbErrorType eb_svt_init_handle(
+EB_API EbErrorType eb_vp9_svt_init_handle(
     EbComponentType** p_handle,
```

Comment:
how about eb_svt_vp9_xxx?

Reply:
> 
> 
> how about eb_svt_vp9_xxx? or simply eb_vp9_xxx?
> @hassount what do you think?

I think those are basically the same:-)
@hassount would you please review this?